packages/stream_chat_flutter/lib/src/message_input/audio_message_send_button/stream_audio_message_overlays.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/widgets.dart';
+import 'package:stream_chat_flutter/stream_chat_flutter.dart';
+
+/// A widget that displays a banner with usage information message.
+class AudioMessageInfoBannerOverlay extends StatelessWidget {
+  /// Returns an [AudioMessageInfoBannerOverlay] with the given [bottomOffset].§
+  const AudioMessageInfoBannerOverlay({
+    super.key,
+    required this.bottomOffset,
+  });
+
+  /// The offset to apply to the overlay.
+  final double bottomOffset;
+
+  @override
+  Widget build(BuildContext context) {
+    final audioRecordingMessageTheme = AudioRecordingMessageTheme.of(context);
+
+    return Positioned(
+      bottom: bottomOffset,
+      child: DefaultTextStyle(
+        style: TextStyle(
+          color: audioRecordingMessageTheme.audioButtonColor,
+          fontSize: 15,
+        ),
+        child: Container(
+          height: 28,
+          width: MediaQuery.of(context).size.width,
+          alignment: Alignment.center,
+          decoration: BoxDecoration(
+            color: audioRecordingMessageTheme.audioButtonBannerColor,
+          ),
+          child: const Text('Hold to start recording.'),
```

Comment:
Is this something we need to support i18n for?

Reply:
Yes, but I want to gather all text by the end of this component, and deal with it in a separate PR after the functionality

Reply:
Okay, makes sense.